minutes/board_2020_10_08.md

```diff
@@ -0,0 +1,73 @@
+# Board Meeting 10/08/2020
+
+## Agenda
+
+### Attendance
+
+*   Adam
+*   Nick
+*   Coty
+
+### Action Items from Previous Meeting
+
+*   Adam
+    *   ~~Create a shared team file on keybase to share passwords (or some other way) Ended up using 1password~~
+    *   Finalize CFP for Natives in Tech Conf 2020 and publicize it
+    *   Setup a meeting for end of Sept to have a strategy meeting about the overall goals of NiT
```

Comment:
Should be something like "Set up a strategy meeting for end of Sept to talk about the overall goals of NiT"

Reply:
Fixed.